overview/dwavesys.rst

```diff
@@ -57,19 +57,19 @@ Client tool installation).
     $ dwave config create
     Configuration file not found; the default location is: C:\\Users\\jane\\AppData\\Local\\dwavesystem\\dwave\\dwave.conf
     Confirm configuration file path (editable):
-    Profile (create new): prod
-    API endpoint URL (editable): https://my.dwavesys.url/
+    Profile (create new):
```

Comment:
The command output changed since the time it was captured for docs.

Now it looks like this (on the first run):
```
$ dwave config create
Configuration file not found; the default location is: /home/radomir/.config/dwave/dwave.conf
Configuration file path [/home/radomir/.config/dwave/dwave.conf]: 
Profile (create new) [prod]: 
API endpoint URL [skip]: 
Authentication token [skip]: asdasdasd
Default client class (qpu or sw) [qpu]: 
Default solver [skip]: 
Configuration saved.
```

Reply:
You removed "Proxy URL"? Thanks, @randomir, I haven't run this for months

Reply:
![image](https://user-images.githubusercontent.com/34041130/64570200-b7ebc780-d314-11e9-9647-d1cf6a576df6.png)